src/main/java/com/rocana/lucene/codec/v1/RocanaLucene50PostingsFormat.java

```diff
@@ -445,9 +445,9 @@ public FieldsConsumer fieldsConsumer(SegmentWriteState state) throws IOException
 
     boolean success = false;
     try {
-      FieldsConsumer ret = new BlockTreeTermsWriter(state, 
+      FieldsConsumer ret = new BlockTreeTermsWriter(state,
                                                     postingsWriter,
-                                                    minTermBlockSize, 
+                                                    minTermBlockSize,
```

Comment:
Should this JIRA do more than just trailing whitespace? Should we make it better conform generally with Rocana coding / formatting standards - such as the indentation on these lines above?